teardown.yaml

```
@@ -53,12 +53,6 @@
       tasks_from: teardown
     # when: "'haproxy' in ansible_facts.packages"
 
-  - name: "Uninstall pip"
-    ansible.builtin.apt:
-      name: python3-pip
-      state: absent
-    when: uninstall_pip
-
   - name: "Remove ununsed packages from the cache"
```

```suggestion
  - name: "Remove unused packages from the cache"
```

I know you didn't touch this, but it hurts my eyes